integrations/jenkins/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "Jenkins"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
we increment it by 0.01 so this should be 0.1.1 the next improvement will be 0.1.2, 0.1.3 in that order

Reply:
@PeyGis  I would have, but I thought this was a new feature, so SemVer recommends incrementing the middle number.

Reply:
Yeah I just read on the SemVer but let's stick with the ocean standard of 0.01. None of our integrations are at 0.2 yet

Reply:
alright then.